docs/cloud/ops/scheduled_backups.md

```diff
@@ -0,0 +1,66 @@
+# Scheduled Backups
+
+Scheduled backups can be created through the [Cloud Console](https://console.eventstore.cloud/) and the [Event Store Cloud CLI](https://github.com/EventStore/esc)
+
+Scheduled backup jobs can be run as frequently as once an hour. After each successful backup, older backups created by the same job will be automatically deleted based on the provided configuration.
+
+::: note
+Multiple scheduled backups can target the same cluster, though if schedules overlap one of the jobs will fail as the cluster will already be locked.
+:::
+For example, you could create one scheduled backup that executes every hour, along with a second scheduled backup that executes once a week. Backups from these scheduled jobs are pruned independently regardless of their age, so if both saved a maximum of four backups, the oldest backup from the weekly job might be close to a month old, while the hourly job's backups would never be older than a fraction of a day.
+
+
+## Console
```

Comment:
Using the Cloud Console

Reply:
Fixed in https://github.com/EventStore/documentation/pull/309/commits/a60d8e8d917d04f339f43d47322231763e0d19dd.